packages/docutils/lib/init.ts

```diff
@@ -140,18 +146,19 @@ export async function initPython({
   upgrade = false,
 }: InitPythonOptions = {}): Promise<void> {
   pythonPath = pythonPath ?? (await findPython()) ?? NAME_PYTHON;
+  const pythonPathWithFlag = [PIP_BREAK_SYSTEM_PACKAGES_FLAG, pythonPath].join(' ');
```

Comment:
this is not really necessary. You may pass custom environment variables using the `env` option of exec

Reply:
Removing this would result in the logging methods not reflecting the use of the environment variable. I can still add the `env` option but that wouldn't make the code any cleaner.

Reply:
I'm not sure this would work at all in Windows. The above syntax where env variables are provided from the command line is only supported by standard *nix shells

Reply:
Thanks, I checked this on Windows and it indeed failed. Changed the code to use exec's `env` option.